ci/unit/test_main.sh

```diff
@@ -7,7 +7,7 @@ set -ex
 # JENKINS-52781 tar function is breaking symlinks
 rm -rf test_results
 mkdir test_results
-# shellcheck disable=SC1091
+# shellcheck source=../../.build_vars.sh
```

Comment:
(lint) Couldn't parse this shellcheck annotation. [SC1073]
(lint) Expected "disable". Fix any mentioned problems and try again. [SC1072]